doc/releases/changelog-dev.md

```diff
@@ -289,6 +289,9 @@
 * Implement `Shots.bins()` method.
   [(#5476)](https://github.com/PennyLaneAI/pennylane/pull/5476)
 
+* Implemented `execute` on `qml.devices.DefaultQutritMixed` device, `execute` can be used to simulate noisy qutrit based circuits.
```

Comment:
Thanks, yes it would be great to have a single shiny changelog entry that shows off `default.qutrit.mixed`, with all the composing PRs listed together.

Reply:
It seems like that was added in a different PR, I moved the message for this PR.